app/models/cart.rb

```diff
@@ -27,6 +27,6 @@ def total_quantity
   end
 
   def total_cost
-    "%.2f" % contents.map {|stache_id, quantity| Stache.find(stache_id).price * quantity}.sum
+    contents.map {|stache_id, quantity| Stache.find(stache_id).price * quantity}.sum
```

Comment:
Line is too long. [84/80]<br>Space between { and | missing.<br>Space missing inside }.